packages/ui/src/components/tag.tsx

```diff
@@ -112,7 +112,7 @@ const Tag = forwardRef<HTMLDivElement, TagProps>(
     return (
       <div
         ref={ref}
-        tabIndex={-1}
+        tabIndex={actionIcon ? -1 : undefined}
```

Comment:
It is not needed here. Focus is needed for the action button, not the entire `Tag`